python/ray/experimental/data/tests/test_dataset.py

```diff
@@ -1141,4 +1141,4 @@ def assert_sorted(sorted_ds, expected_rows):
 
 if __name__ == "__main__":
     import sys
-    sys.exit(pytest.main(["-v", __file__, "-k", "sort"]))
```

Comment:
btw, one thing i do is:

sys.exit(pytest.main(["-v", __file__] + sys.argv[1:])
so i don't need to edit source to add flags

Reply:
You can just do `pytest -v -s test_dataset.py::sort`; no need to actually run the script main.